gazelle/js/typescript/tsconfig.go

```diff
@@ -99,6 +118,8 @@ func parseTsConfigJSON(cm *TsConfigMap, root, configDir string, tsconfigContent
 	var BaseUrl string
 	if c.CompilerOptions.BaseUrl != nil {
 		BaseUrl = path.Clean(*c.CompilerOptions.BaseUrl)
+	} else if baseConfig != nil && baseConfig.BaseUrl != "" {
```

Comment:
Does one of your tests fail without this?

Reply:
No, but it handles the case, if a config was extended with a baseUrl and the local one not having one the extended one will be used. It's quite difficult to test this case without "a real file", because the parsing function loads the extension of a file directly. So we can remove this but then the case with a baseDir inherited from an extension file will not be covered.

Reply:
How should I handle this? Just leave it as is?